swift_package/BrainFlowCI/BrainFlowCI.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
```

Comment:
and this

Reply:
I am not sure. I can try removing it from the repo and see what happens.

Reply:
git rm --cached but i made a backup first just in case